components/loginPage.js

```diff
@@ -1,37 +1,51 @@
 import React, { useState } from 'react';
-import { StyleSheet, TextInput, View, TouchableOpacity, Text } from 'react-native';
+import { StyleSheet, TextInput, View, TouchableOpacity, Text, AsyncStorage } from 'react-native';
 import OAuth from './oauth';
+import axios from 'axios';
 
 function LoginPage(props) {
   let [email, setEmail] = useState('');
   let [password, setPassword] = useState('');
+  let [errorText, setErrorText] = useState('');
 
   return (
     <View style={styles.container}>
       <View style={styles.logo}>
         <Text>LOGO</Text>
       </View>
       <TextInput style={styles.textInput} placeholder="Email" onChangeText={text => setEmail(text)} />
-      <TextInput style={styles.textInput} placeholder="Password" onChangeText={text => setEmail(text)} />
+      <TextInput style={styles.textInput} placeholder="Password" onChangeText={text => setPassword(text)} />
+      <Text style={styles.error}>{errorText}</Text>
 
       <TouchableOpacity
         onPress={() => {
-          // go to sign up page
+          axios
+            .post('http://localhost:8080/login', { email, password })
+            .then(async response => {
+              setErrorText('');
```

Comment:
Odd you have to set error text again here? Unless it is to clear previous errors